WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -846,7 +848,12 @@ private void onAppComesFromBackground() {
                 // Refresh account informations and Notifications
 
                 if (AccountHelper.isSignedInWordPressDotCom()) {
-                    NotificationsUpdateService.startService(getContext());
+                    if (activity.getIntent().hasExtra(NotificationsListFragment.NOTE_ID_EXTRA)) {
```

Comment:
While theoretically possible, I think it's quite tricky for it to happen here, anyway good catch addressed in fee0f38